spec/unit/local_system_spec.rb

```diff
@@ -133,10 +133,12 @@
 
       LocalSystem.validate_machinery_compatibility
 
+      supported_oses = Os.supported_host_systems.map(&:canonical_name).sort.join(", ")
+      expect(captured_machinery_output).to include(supported_oses)
       expect(captured_machinery_output).to include(
-        ": SUSE Linux Enterprise Server 12, openSUSE 13.1 (Bottle)," \
```

Comment:
I would check not just for the variable but at least some operating systems which we will be supporting for quite some time, like SLES12 and 13.2